src/probe_modules/module_tcp_synscan.c

```diff
@@ -16,22 +16,60 @@
 #include <assert.h>
 
 #include "../../lib/includes.h"
+#include "../../lib/logger.h"
 #include "../fieldset.h"
+#include "module_tcp_synscan.h"
 #include "probe_modules.h"
 #include "packet.h"
 #include "validate.h"
 
-#define ZMAP_TCP_SYNSCAN_TCP_HEADER_LEN 24
-#define ZMAP_TCP_SYNSCAN_PACKET_LEN 58
+// defaults
+static uint8_t ZMAP_TCP_SYNSCAN_TCP_HEADER_LEN = 20;
```

Comment:
Given that these aren't constants, I don't think they should be capitalized

Reply:
Good call!